### ISSUE TITLE
monoidal-containers is very old

### ISSUE THREAD

**Issue by dfordivam** [obsidiansystems/rhyolite#108]
Its one year old and more importantly does not have instance of `Align (MonoidalMap k)`

**Comment by dfordivam** [comment]
I see this being bumped in #99 and #95 gets rid of pinned version.

Though after doing the bump to 0.5.0.0 to include the Align instances (https://github.com/bgamari/monoidal-containers/commit/df270e82d09be1b988b0a4e564f2b34b4174d291) I got errors in vessel

Neither of these PRs mention/handle this error, so I am not sure how they managed to compile

```
src/Data/Vessel.hs:572:39: error:
    • Could not deduce (Semigroup a) arising from a use of ‘Map.insert’
      from the context: Ord k
        bound by the type signature for:
                   findPivot :: forall k a. Ord k => MonoidalMap k a -> Pivot k a
        at src/Data/Vessel.hs:567:1-50
      Possible fix:
        add (Semigroup a) to the context of
          the type signature for:
            findPivot :: forall k a. Ord k => MonoidalMap k a -> Pivot k a
    • In the second argument of ‘Split’, namely ‘(Map.insert k v l)’
      In the expression: Split k (Map.insert k v l) r
      In a case alternative:
          [(k, v)]
            | Map.null l && Map.null r -> One k v
            | otherwise -> Split k (Map.insert k v l) r
    |
572 |               | otherwise -> Split k (Map.insert k v l) r
```

**Comment by dfordivam** [comment]
aah so v0.6.0.0 reverted the semigroup stuff, now I understand.

**Comment by dfordivam** [comment]
This has been fixed with #95
